test/component/FileUploadSchemaSpec.scala

```diff
@@ -63,6 +64,23 @@ class FileUploadSchemaSpec extends ComponentTestSpec
       headers(result).get(X_CONVERSATION_ID_NAME) shouldBe 'defined
     }
 
+    scenario("Response status 400 when user submits a malformed xml payload") {
```

Comment:
I think this test should be merged with FileUploadSpec, plus it is not exactly schema spec, it is more like unhappy path specs

Reply:
will fix